common/src/main/java/com/example/android/uamp/media/UampNotificationManager.kt

```diff
@@ -0,0 +1,113 @@
+package com.example.android.uamp.media
+
+import android.app.PendingIntent
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.net.Uri
+import android.support.v4.media.session.MediaControllerCompat
+import android.support.v4.media.session.MediaSessionCompat
+import com.google.android.exoplayer2.ExoPlayer
+import com.google.android.exoplayer2.Player
+import com.google.android.exoplayer2.ui.PlayerNotificationManager
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.SupervisorJob
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+
+const val NOW_PLAYING_CHANNEL: String = "com.example.android.uamp.media.NOW_PLAYING"
+const val NOW_PLAYING_NOTIFICATION: Int = 0xb339
```

Comment:
Same as above. Also is the number arbitrary?

Reply:
@nic0lette Do you know why this number was chosen?

Reply:
Yes, I picked it :) It's completely arbitrary though.

Reply:
Added comment